dependencies.gradle

```diff
@@ -33,7 +33,8 @@ ext {
   }
 
   // TODO: Remove when we remove the old backend.
-  makeWarningsErrors = ci && kotlinUseIR
+  // TODO Repeatable: After cleaning up all deprecated methods remove the `&& false`.
+  makeWarningsErrors = ci && (kotlinVersion.startsWith('1.5') || kotlinUseIR) && false
```

Comment:
I already did the same in the large PR where I started implementing repeatable annotations.